azure-boards-common/src/main/java/com/synopsys/integration/azure/boards/common/service/workitem/AzureWorkItemService.java

```diff
@@ -124,10 +91,8 @@ public WorkItemResponseModel updateWorkItem(String organizationName, String proj
     }
 
     public WorkItemDeletedResponseModel deleteWorkItem(String organizationName, Integer workItemId) throws HttpServiceException {
-        String requestSpec = API_SPEC_ORGANIZATION_PROJECT_WORKITEMS_INDIVIDUAL
-                                 .defineReplacement(PATH_ORGANIZATION_REPLACEMENT, organizationName)
-                                 .defineReplacement(PATH_WORK_ITEM_ID_REPLACEMENT, workItemId.toString())
-                                 .populateSpec();
+        //TODO verify this is a bug
```

Comment:
Looks like this was supposed to be based on: `API_SPEC_ORGANIZATION_WORKITEMS_INDIVIDUAL`

Reply:
I used API_SPEC_ORGANIZATION_PROJECT_WORKITEMS_INDIVIDUAL because the spec template looked to be the same for getting the WorkItemDeleted. I realized it matched the ...WORKITEMS_INDIVIDUAL so I reused it in this spot

Reply:
Right, but that would not have worked right?

Reply:
@ekerwin This is a bug. Project name is optional in the request, but cannot be null.

Reply:
Thanks @gkillough !